server/database/queries/signUpUser.ts

```diff
@@ -0,0 +1,31 @@
+import userSchema from '../models/user';
+
+const createAccount = async ({
+  email, hashedPassword, name, role, accessToken,
+} : { email: string, hashedPassword: string, name: string,
+  role: string, accessToken: string }) => {
+  const password: string = hashedPassword;
+  const user = await userSchema.create({
+    email, password, name, role, accessToken,
+  });
+  return user;
+};
+
+const findAccount = async (data: object) => {
+  const user = await userSchema.findOne({ data });
+  return user;
+};
+
+const updateValidate = async (data: object, check:boolean) => {
+  const updated = await userSchema.updateOne({ data }, { $set: { is_verified: check } });
+  return updated;
+};
+
+const deleteAccount = async (data: object) => {
+  const user = await userSchema.deleteOne({ data });
+  return user;
+};
```

Comment:
They are all methods so when need them just use them in your controller, you don't need to separate them. We don't need this folder